packages/theme/src/cli/commands/theme/rename.ts

```diff
@@ -0,0 +1,50 @@
+import ThemeCommand from '../../utilities/theme-command.js'
+import {RenameOptions, renameTheme} from '../../services/rename.js'
+import {ensureThemeStore} from '../../utilities/theme-store.js'
+import {themeFlags} from '../../flags.js'
+import {Flags} from '@oclif/core'
+import {globalFlags} from '@shopify/cli-kit/node/cli'
+import {ensureAuthenticatedThemes} from '@shopify/cli-kit/node/session'
+
+export default class Rename extends ThemeCommand {
+  static description = 'Renames an existing theme.'
+
+  static flags = {
```

Comment:
May we support the `--live` flag as well? In the other commands, we generally support `-d`/`-t`/`-l` for theme selection :)